src/toolchain/mod.rs

```diff
@@ -215,11 +215,16 @@ pub async fn install(args: InstallOpts, install_mode: InstallMode) -> Result<()>
             let xtensa_gcc = Gcc::new(XTENSA_GCC, &host_triple, &toolchain_dir);
             to_install.push(Box::new(xtensa_gcc));
         }
+
         // All RISC-V targets use the same GCC toolchain
         // ESP32S2 and ESP32S3 also install the RISC-V toolchain for their ULP coprocessor
```

Comment:
This sentence would no longer be true, I'd remove line 221 so there is not an empty line between the to comments and clarify that ESP32S2 and ESP32S3 can also opt-in to install RISC-V GCC toolchain, which is used for their ULP coprocessor

Reply:
updated comments